shinyTools 24.12.0: update links and logos in headerUI

arunge opened the pull request:
# shinyTools 24.12.0

## Updates
- _headerUI_: integration of new links, and logos loaded from the https://github.com/Pandora-IsoMemo/docs/blob/main/img/ folder

arunge commented:
@f-lukas Could you please review this PR?

f-lukas commented:
I checked the changes with the example app and it looked like this. Does it look the same for you?
![Bildschirmfoto vom 2024-12-18 08-59-53](https://github.com/user-attachments/assets/26133fec-fda4-4831-a16c-6dadc12d0b7f)

I opened MapR, installed the shinyTools version from the feature branch but could not see the header at all. I have then copied the css file from shinyTools and have added it to MapR. Now the header looked as expected with all the new logos.

So I think your code changes are totally fine but there is an issue with `shinyTools::includeShinyToolsCSS()` :(

arunge commented:
@f-lukas For me the testapp looks like this ... :thinking: 

![image](https://github.com/user-attachments/assets/172220c6-4f38-4297-a978-8d64bd7675c3)

I will check the header in another app... and see how to solve the issue with `shinyTools::includeShinyToolsCSS()`

arunge commented:
Regarding

> So I think your code changes are totally fine but there is an issue with shinyTools::includeShinyToolsCSS() :(

I had also issues with loading the css in other apps. It occurred that the resourcePath that we are setting in shinyTools was overwritten, and the wrong css was loading. I changed the name "app_files" to a more specific one: "shinyTools_files".
Additionally, I removed links to github for loading logos and now load local logos.

@f-lukas Could you please check if all is working in your environment also?

f-lukas reviewed (approved):
Perfect, it works for MapR and also the example app.
